usr/share/rear/prep/BORG/default/100_set_vars.sh

```diff
@@ -13,8 +13,13 @@ borg_set_vars
 # combined string of "BORGBACKUP_USERNAME@BORGBACKUP_HOST".
 # borg_dst_dev directory will be created in later stage
 # (if not already present) by 250_mount_usb.sh script.
+
 if [[ -n $BORGBACKUP_HOST ]]; then
-    borg_dst_dev=$BORGBACKUP_USERNAME@$BORGBACKUP_HOST:
-else
-    borg_dst_dev=$BUILD_DIR/borg_backup
+   borg_dst_dev=$BORGBACKUP_USERNAME@$BORGBACKUP_HOST:
+   # We have to test if anoher port is used.
+      if [[ -n $BORGBACKUP_PORT ]]; then
```

Comment:
Please get rid of this condition, and define _BORGBACKUP_PORT=22_ in [default.conf](https://github.com/rear/rear/blob/master/usr/share/rear/conf/default.conf) instead.
That will provide reasonable default value that will not need explicit check.

Reply:
yes, it's a possibility but you need to use this syntax :
```
if [[ -n $BORGBACKUP_HOST ]]; then
   borg_dst_dev=ssh://$BORGBACKUP_USERNAME@$BORGBACKUP_HOST:$BORGBACKUP_PORT/
else
borg_dst_dev=$BUILD_DIR/borg_backup
fi
```

Reply:
I don't mind the new style (ssh:// prefix) in `borg_dst_dev=ssh://$BORGBACKUP_USERNAME@$BORGBACKUP_HOST:$BORGBACKUP_PORT` ...
Just remove the condition and define _BORGBACKUP_PORT=22_ in default conf ;-)